packages/footer/src/components/Footer.js

```diff
@@ -52,6 +53,7 @@ Footer.propTypes = {
   helpText: PropTypes.string.isRequired,
   helpLinkText: PropTypes.string,
   helpLinkHref: PropTypes.string,
+  helpLinkOnClick: PropTypes.func,
```

Comment:
maybe this name can be ``onHelpLinkClick`` to follow the others: ``onSecondaryActionClick`` and ``onPrimaryActionClick``

Reply:
I came up with `helpLinkOnClick` so keep the prefix of the other props. But sure, i'll change that